engine/lite/request-response/runtime/src/test/scala/pl/touk/nussknacker/engine/requestresponse/RequestResponseInterpreterSpec.scala

```diff
@@ -105,7 +105,7 @@ class RequestResponseInterpreterSpec extends FunSuite with Matchers with Patient
     val process = EspProcessBuilder
       .id("proc1")
       .source("start", "request1-post-source")
-      .customNode("for-each", "outPart", "for-each", "elements" -> "#input.toList()")
+      .customNode("for-each", "outPart", "for-each", "Elements" -> "#input.toList()")
```

Comment:
can we do it without migration ?

Reply:
we already changed component name... it's not (widely) used in production...

Reply:
that's what I hoped for, the component is only available in lite engine, that has not been released yet.